product-management/Purpose of MS Team Channels.md

```diff
@@ -0,0 +1,140 @@
+# Overview
```

Comment:
The content in this doc is largely accurate, its too long. The problem with that is people would tend to overlook it. We should try to say the same but in 20% of words used in this doc, without leaving key details.